types/algoliasearch.d.ts

```diff
@@ -0,0 +1,124 @@
+// Note: Below, we will be importing all algoliasearch v3,4,5 types.
+// The goal is being able to export the algoliasearch-helper types using
+// the version of the client installed by the developer.
+
+// @ts-ignore
+import type algoliasearch from 'algoliasearch/lite';
+// @ts-ignore
+import type * as AlgoliaSearchLite from 'algoliasearch/lite';
+// @ts-ignore
+import type * as ClientSearch from '@algolia/client-search';
+// @ts-ignore
+import type * as AlgoliaSearchLiteV5 from '@experimental-api-clients-automation/algoliasearch-lite';
+// @ts-ignore
+import type * as AlgoliaSearchFullV5 from '@experimental-api-clients-automation/algoliasearch';
+// @ts-ignore
+import * as ClientSearchV5 from '@experimental-api-clients-automation/client-search';
```

Comment:
these get renamed to @algolia at public beta, so will probably be able to be merged into the other imports